### PR TITLE
[HUD][CH] api/.../commit and api/.../pull to only use ClickHouse

### PR DESCRIPTION
**Pull request by clee2000**
Similar to https://github.com/pytorch/test-infra/pull/6034 but for api/.../commit and api/.../pull

### CODE REVIEW

File: torchci/lib/fetchCommit.ts [hunk 1]
```diff
@@ -1,66 +1,35 @@
...
-            name: "repo",
-            type: "string",
-            value: `${owner}/${repo}`,
-          },
-        ],
-      }
-    );
-    return response.results!;
+  for (const row of response) {
+    // TODO: change the code that relies on this logic?
```
Comment (ZainRizvi) [inline torchci/lib/fetchCommit.ts:15]: Could you elaborate more here? Change the code to do what?

File: torchci/pages/[repoOwner]/[repoName]/pull/[prNumber].tsx [hunk 2]
```diff
@@ -24,9 +24,7 @@ function CommitInfo({
 }) {
   const useCH = useCHContext().useCH;
```
Comment (ZainRizvi) [inline torchci/pages/[repoOwner]/[repoName]/pull/[prNumber].tsx:25]: unnecessary `useCh`

**Review by ZainRizvi — APPROVED** [review]

File: torchci/lib/fetchCommit.ts (hunk 1 above)
Comment (clee2000) [reply torchci/lib/fetchCommit.ts]: Change the code that looks for null to look for 0 instead.  Also edited comment to reflect this

**Review by huydhn — APPROVED** [review]
LGTM! Also I just remember `yarn lint` might come into handy for this kind of changes, but take it with a grain of salt.

**Pull request merged**